app/assets/stylesheets/bootstrap_and_customization.css.scss

```diff
@@ -63,10 +63,63 @@ img {
   font-weight: bold;
 }
 
+.navbar .navbar-default {
+  height: 80px;
+}
+
+.bs-docs-container,
+.bs-docs-section {
+  padding-top: 77px;
+}
+
+.video-panel {
+  min-height: 369px;
+}
+
+footer {
+  box-shadow: 0px -2px 4px rgba(194, 194, 194, 0.44);
+  margin-top: 25px;
+  padding-bottom: 10px;
+}
+
+a.btn.btn-default {
+  background-color: rgba(255, 255, 255, 0.3);
+  color: gray;
+  &:hover {
+    color: #FFF;
+    background-color: #019C91;
+    border-color: #019C91;
```

Comment:
Color literals like `#019C91` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#019C91` should be written as `#019c91`